### DISCUSSION TITLE
Account flagged

### DISCUSSION THREAD

**Discussion by smartrittik** [community/community#31756] [Other Feature Feedback, Questions, & Ideas] [answered]
My account is flagged, and I don't know the reason, some are saying it's due to multiple account in the same device, but I don't have one.
So please github look after this problem, there are many important accounts linked to my github like gitcoin guild whose access is lost due to this reason, please help me out.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @smartrittik! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :) The Community is unable to assist in matters like these so we'll be locking the thread.
